### PR TITLE
Fix/credentials

### LINKED ISSUE #623: Fix account recovery components
- [x] Research how other apps show similar components
- [x] Component needs to be removed from sidebar menu
- [x] Component must appear as an option "Forgot password? // Forgot username?" when trying to login.
- [x] Component needs to less crammed -- either the user wants to recover password o username. 

Captura de Pantalla 2021-03-31 a la(s) 09 47 56

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #623 

### What does this PR do?
Credentials Recovery segmentation, password recovery
image

### How should this be tested?
- Make run
- Go to Login
- Recover the password

### CODE REVIEW

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 1]
```diff
@@ -237,20 +223,13 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
-        <Link to="/">
-          <Typography
-            variant="body1"
-            className={clsx(classes.labelOption, overrideLabelClass)}
-          >
-            {t('credentialsRecovery.credentialsRecovery')}
-          </Typography>
-        </Link>
-      </Box>
+
```
Comment (JustinCast) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:226]: Delete this blank space

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 2]
```diff
@@ -237,20 +223,12 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
-            {t('credentialsRecovery.credentialsRecovery')}
-          </Typography>
-        </Link>
-      </Box>
+        <Box className={classes.recoveryBox}>
+          <Button color="secondary" className={classes.recoveryButton} onClick={handleOpen}>
+          {t('signup.forgetPassword')}
+          </Button>
+        </Box>
+
```
Comment (JustinCast) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:231]: Delete blank space here

**Review by JustinCast — COMMENTED** [review]
Take a look at the conflicts!

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
I think there is too much space at the bottom, we can move the recover button a little bit lower

image

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
@Edgar097 take a look at this warning please
`Check warning on line 113 in webapp/src/components/CredentialsRecovery/CredentialsRecovery.js`

And also 
image

Both of them are in CredentialsRecovery.js

**Review by JustinCast — APPROVED** [review]

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 3]
```diff
@@ -198,32 +196,23 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
         setErrorMessage(t('credentialsRecovery.emailError'))
       else setErrorMessage(error.message.replace('GraphQL error: ', ''))
     }
-  }, [error])
+  }, [error,t])
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:199]: add space after comma

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 4]
```diff
@@ -198,32 +196,23 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
 
   useEffect(() => {
     if (errorChangePassword) {
       if (errorChangePassword.message === `GraphQL error: Cannot read property 'secret' of null`)
         setErrorMessage(t('credentialsRecovery.emailError'))
       else setErrorMessage(errorChangePassword.message.replace('GraphQL error: ', ''))
     }
-  }, [errorChangePassword])
+  }, [errorChangePassword,t])
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:207]: add space after comma

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Minor changes

**Review by steph222 — APPROVED** [review]

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 5]
```diff
@@ -116,10 +113,10 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
   ] = useMutation(CREDENTIALS_RECOVERY)
   const [
     changePassword,
-    { loading: loadingChangePassword, error: errorChangePassword, data: { change_password: responseChangePassword } = {} }
+    { loading: loadingChangePassword, error: errorChangePassword }
   ] = useMutation(CHANGE_PASSWORD)
   const [open, setOpen] = useState(false)
-
+
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:119]: Leave the line break but remove the black space

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 6]
```diff
@@ -45,24 +43,25 @@ const useStyles = makeStyles((theme) => ({
...
+    lineHeight: 1.33,
+    letterSpacing: '0.4px',
+    color: '#000000'
   },
-  labelOption: {
-    color: theme.palette.primary.main,
-    marginLeft: theme.spacing(3),
-    fontSize: 14,
-    cursor: "pointer"
+  recoveryBox: {
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:55]: Remove empty space at the end

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Minor changes

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 7]
```diff
@@ -308,132 +288,29 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
                 >
                   {t('credentialsRecovery.recovery')}
                 </Button>
+                <Box className={classes.recoveryBox}>
                 {loading && <CircularProgress />}
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:292]: Make indentation

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 8]
```diff
@@ -308,132 +288,29 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
-                  className={classes.button}
-                >
-                  {t('credentialsRecovery.changePassword')}
-                </Button>
-                {loadingChangePassword && <CircularProgress />}
+                </Box>
               </Box>
               {errorMessage && (
+                <Snackbar open={true} autoHideDuration={4000} onClose={handleCloseSnackBar}>
                 <Alert
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:297]: Make indentation

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 9]
```diff
@@ -308,132 +288,29 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
-                  }
-                >
-                  {t('credentialsRecovery.errorPassword')}
-                </Alert>
+                </Alert>                  
+                </Snackbar>
               )}
               {success && (
+                 <Snackbar open={success} autoHideDuration={4000} onClose={handleCloseSnackBar}>
                 <Alert
```
Comment (leisterfrancisco) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:307]: Make indentation

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Minor indentations errors

**Comment by leisterfrancisco** [comment]
@Edgar097 take a look at merge conflicts

**Comment by steph222** [comment]
> @Edgar097 take a look at merge conflicts

@Edgar097  Look this

**Comment by Edgar097** [comment]
> > @Edgar097 take a look at merge conflicts
> 
> @Edgar097 Look this

@leisterfrancisco @steph222 can you check if it is working please

**Comment by leisterfrancisco** [comment]
> > > @Edgar097 take a look at merge conflicts
> > 
> > 
> > @Edgar097 Look this
> 
> @leisterfrancisco @steph222 can you check if it is working please

@Edgar097 is working perfectly, check your local changes

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**